tests/testuser

```diff
@@ -0,0 +1,2 @@
+# test version of developers file
```

Comment:
feel like this file could have a better name - `testdevelopers`? `developers-test`? etc?

Reply:
tests/developers?